### ISSUE TITLE
How to approach testing?

### ISSUE THREAD

**Issue by davish** [hotwire-django/turbo-django#3]
I'm wondering how to approach testing for the utilities we're building? Probably most can be unit-tested, but it's probably also a good idea to make sure they work as intended with Turbo. We could set up integration tests with [Cypress](https://www.cypress.io)?

**Comment by blopker** [comment]
Yes! Having and end-to-end test or two will really help. We could run it against one of the sample apps in the repo. 

I've had teams in the past use Cypress and have really liked it. However, I wonder if it's too much machinery for this project since Turbo's interactions are pretty simple. Maybe something like https://playwright.dev/ would be simpler to set up in CI. The Docker story seems more straightforward: https://playwright.dev/docs/docker/README. I also think we can get away with only testing in 1 browser since all the heavier testing would be done in the official Turbo repo.

**Comment by blopker** [comment]
Although, the Cypress Github Actions support looks pretty good! https://github.com/cypress-io/github-action

**Comment by jonathan-s** [comment]
It's not difficult at all to set up tests with cypress. I'm using cypress for sockpuppet.
